scripts/cut_release_or_hotfix_branch.py

```diff
@@ -166,29 +214,51 @@ def _execute_branch_cut():
     common.open_new_tab_in_browser_if_possible(
         'https://github.com/oppia/oppia#oppia---')
     while True:
+        if not HOTFIX_VERSION:
+            branch_to_check = 'develop'
+        elif HOTFIX_VERSION == 1:
+            branch_to_check = 'release-%s' % TARGET_VERSION
+        else:
+            branch_to_check = 'release-%s-hotfix-%s' % (
+                TARGET_VERSION, HOTFIX_VERSION - 1)
         python_utils.PRINT(
-            'Please confirm: are Travis checks passing on develop? (y/n) ')
+            'Please confirm: are Travis checks passing on %s? (y/n) ' % (
+                branch_to_check))
         answer = python_utils.INPUT().lower()
         if answer in release_constants.AFFIRMATIVE_CONFIRMATIONS:
             break
         elif answer:
             python_utils.PRINT(
-                'Tests should pass on develop before this script is run. '
-                'Exiting.')
+                'Tests should pass on %s before this script is run. '
+                'Exiting.' % branch_to_check)
             sys.exit()
 
     # Cut a new release branch.
-    python_utils.PRINT('Cutting a new release branch: %s' % NEW_BRANCH_NAME)
-    subprocess.call(['git', 'checkout', '-b', NEW_BRANCH_NAME])
+    if HOTFIX_VERSION:
+        _verify_hotfix_version_is_one_ahead_of_previous_hotfix_version(
+            remote_alias)
+        new_branch_type = 'hotfix'
```

Comment:
This can probably set way earlier, and instead of doing if HOTFIX_VERSION, we can probably check the new_branch_type.

Reply:
Done.